sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -1072,8 +1068,9 @@ private UUID saveErrorLogFiles(Throwable throwable, ManagedErrorLog errorLog) th
         File throwableFile = new File(errorStorageDirectory, filename + ErrorLogHelper.THROWABLE_FILE_EXTENSION);
         if (throwable != null) {
             try {
-                FileManager.writeObject(throwableFile, throwable);
-                AppCenterLog.debug(Crashes.LOG_TAG, "Saved Throwable as is for client side inspection in " + throwableFile + " throwable:", throwable);
+                String stackTrace = android.util.Log.getStackTraceString(throwable);
+                FileManager.write(throwableFile, stackTrace);
+                AppCenterLog.debug(Crashes.LOG_TAG, "Saved stackTrace as is for client side inspection in " + throwableFile + " stackTrace:" + stackTrace);
```

Comment:
Please remove `Crashes.`
```suggestion
                AppCenterLog.debug(LOG_TAG, "Saved stackTrace as is for client side inspection in " + throwableFile + " stackTrace:" + stackTrace);
```